lib/rust/launcher-shims/src/lib.rs

```diff
@@ -0,0 +1,78 @@
+use std::env;
+use std::fs::OpenOptions;
+use std::io;
+use std::io::prelude::*;
+use std::path::PathBuf;
+use std::process::{Command, exit};
+
+
+
+// ====================
+// === WrapLauncher ===
+// ====================
+
+/// Runs the wrapped launcher overriding its reported version to the provided version.
```

Comment:
We use imperative style in our rust doc comments, so this should be "Run the wrapped launcher..."